adr/2023-03-28-towercrush-highscore.md

```diff
@@ -0,0 +1,19 @@
+# Highscore of towercrush
+
+## Possible Solutions
+
+- a: after winning a game once show 100%, else 0%
+- b: calculate % of correct answered questions, respect if won
+- c: calculate % of correct answered questions, DONT respect if won
+
+## Chosen Solution
+
+a 
+
+## Pro Chosen Solution
+
+easy to implement, easy to understand.
+
+## Contra Chosen Solution
+
+--
```

Comment:
```suggestion
 No approach works when only a single team is left.
 In this case, either the lecturer should be the opposing team, or another team might be so gracious as to let the other students pass.
```

Reply:
not true, you can play the game allone answer one question right and you pass

Reply:
I thought you need two teams to play?
How can you play alone?